themes/prologue/prologue.rst

```diff
@@ -2,22 +2,20 @@
 Prologue
 ********
 
-The default PrestaShop install offers a neutral theme in black and white, enabling sellers to quickly and freely start their activity, whatever their business line.
+The default PrestaShop installation comes with one simple theme styled in black and white. This neutral style allows any seller to begin selling right away.
 
-Nearly 2000 themes are available through the PrestaShop Addons marketplace (http://addons.prestashop.com/en/). They were created either by members of the PrestaShop community or the PrestaTeam, and are sold at reasonable prices – some are even free.
+There are over 2,000 themes available through the [**PrestaShop Addons Marketplace**] (http://addons.prestashop.com/en/). Created by members of the PrestaShop community or the PrestaTeam. And, sold at reasonable prices – some are even free.
```

Comment:
While I'm okay with cutting long sentences in two (or more) and starting sentences with a conjunction, the whole 'follow a conjunction with a comma' thing here is a big no-no to me. I agree with [the best answer here](http://english.stackexchange.com/a/91647): "a comma is a “disjunction”, and defeats the purpose of the conjunction". Please update the PR accordingly.